src/egon/data/datasets/pypsaeursec/__init__.py

```diff
@@ -670,26 +670,53 @@ def lines_to_etrago(neighbor_lines=neighbor_lines, scn="eGon100RE"):
     lines_to_etrago(neighbor_lines=neighbor_lines, scn="eGon2035")
 
     # prepare and write neighboring crossborder links to etrago tables
-    def links_to_etrago(neighbor_links, scn="eGon100RE"):
+    def links_to_etrago(neighbor_links, scn="eGon100RE", extendable=True):
         neighbor_links["scn_name"] = scn
 
-        for i in [
-            "name",
-            "geometry",
-            "tags",
-            "under_construction",
-            "underground",
-            "underwater_fraction",
-            "bus2",
-            "bus3",
-            "bus4",
-            "efficiency2",
-            "efficiency3",
-            "efficiency4",
-            "lifetime",
-            "p_nom_opt",
-        ]:
-            neighbor_links = neighbor_links.drop(i, axis=1)
+        if extendable == True:
+            for i in [
+                "name",
+                "geometry",
+                "tags",
+                "under_construction",
+                "underground",
+                "underwater_fraction",
+                "bus2",
+                "bus3",
+                "bus4",
+                "efficiency2",
+                "efficiency3",
+                "efficiency4",
+                "lifetime",
+                "p_nom_opt",
+                # "pipe_retrofit",
```

Comment:
Could this line be deleted? It is a bit confusing to have commented lines in the code.

Reply:
I intentionnaly left this commented line, because by running the code with latest p-e-s results (not in the databundle but on the server), this new column was creating error in the insertion of the table in the database. But it could be added later on, so I could delete it.

Reply:
Thanks for the advise, I've changed that in https://github.com/openego/eGon-data/pull/885/commits/25594a5a8c13b2ac83f63cfa051be1976abdb8e3.